examples/OpenSLL-asm/config.cmake

```diff
@@ -0,0 +1 @@
+hunter_config(OpenSSL VERSION 1.0.2p CMAKE_ARGS ASM_SUPPORT=ON)
```

Comment:
Some mess with the examples... Now I see `examples/OpenSLL-asm` and `examples/OpenSSL-asm`.